srcpkgs/kgpg/template

```diff
@@ -18,4 +20,11 @@ license="GPL-2.0-only"
 homepage="https://kde.org/applications/en/utilities/org.kde.kgpg"
 changelog="https://kde.org/announcements/changelogs/gear/${version}/#kgpg"
 distfiles="${KDE_SITE}/release-service/${version}/src/${pkgname}-${version}.tar.xz"
-checksum=4de7b4b2879f2ae5cbcb3152246e607304aa4dac1da875b1da491908cf9100f7
+checksum=7504ba190ec2385bebd2dea06d0b325cc5fe9a4cb5a5f9a5cff8a961acdfa28b
+
+do_check() {
```

Comment:
either overwrite `do_check` or `make_cmd`/`make_check_args`, not both at the same time

Reply:
fixed